README.md

```diff
@@ -60,4 +60,28 @@ In order to run the full suite of Acceptance tests, run `make testacc`.
 
 ```sh
 $ make testacc
+# run a single acceptance test: e.g. 'TestAccDockerRegistryImage_private' in 'data_source_docker_registry_image_test.go'
+go test -v -timeout 30s github.com/terraform-providers/terraform-provider-docker/docker -run ^TestAccDockerRegistryImage_private$
 ```
+
+In order to extend the provider and test it with `terraform`, check the latest version at https://releases.hashicorp.com/terraform-provider-$PROVIDER_NAME/ and raise it accordingly to semversion due to your changes in the code
+```sh
+$ export TF_PROVIDER_NEW_VERSION=0.1.2
+$ go build -o terraform-provider-${PROVIDER_NAME}_v${TF_PROVIDER_NEW_VERSION}
+# create ~/.terraform.d/plugins/'GOOS'_'GOARCH' with
+# GOOS: darwin, freebsd, linux, and so on.
+# GOARCH: 386, amd64, arm, s390x, and so on
+# example for OSX:
+$ mkdir -p ~/.terraform.d/plugins/darwin_amd64
```

Comment:
I'm not sure what the above is instructing users to do... 
> In order to extend the provider and test it with
Do you mean in order to work with this code _as is_, or develop? Shouldn't we ask users to clone this repo and then use `make`?  Perhaps I'm confused at the intent of these in instructions 🤔

Reply:
I didn't understand how to raise the version of the docker provider when developing locally. Though about raising a version number like I know from `npm` or `java` libs. I looked at the `terraform` code how plugins are fetched and ended up with the solution mentioned above. Fortunately, I saw your [talk](https://www.youtube.com/watch?v=TMmovxyo5sY&t=30m14s) about `Terraform and The Extensible Provider Architecture` yesterday where you explain how it should be done 😃  I'll adapt it to a better explanation how to develop locally.